src/ccnl-core/src/ccnl-prefix.c

```diff
@@ -595,29 +595,29 @@ ccnl_prefix_to_str_detailed(struct ccnl_prefix_s *pr, int ccntlv_skip, int escap
     (void)j;
     (void)len;
     (void) call_slash;
-    (void)buflen; //fixme, prevent bufferoverrun here!
+    (void)buflen;
```

Comment:
I assume that ``buflen`` was void'ed here since the parameter was not used in the function before. However, the ``buflen`` parameter is used in the function and hence the statement (which silenced the "unused parameter ..." compiler warning) can be removed.

Or is there a IFDEF branch where ``buflen`` and ``len`` are not used?

Reply:
I will remove that